cmake/python.cmake

```diff
@@ -4,7 +4,7 @@
 
 # Try to find version 3.  If not, we fall back to version 2.
 find_package(PythonInterp QUIET 3)
-find_package(PythonInterp QUIET)
+find_package(PythonInterp QUIET 2.4)
```

Comment:
It would be completely reasonable to set this to 2.7. Anything older is going to be problematic for a bunch of other reasons, even if it does compile.

Reply:
It's just what we had before.